usr/src/cmd/truss/codes.c

```diff
@@ -2032,6 +2033,8 @@ openarg(private_t *pri, int arg)
 		(void) strlcat(str, "|O_DIRECT", sizeof (pri->code_buf));
 	if (arg & FXATTRDIROPEN)
 		(void) strlcat(str, "|FXATTRDIROPEN", sizeof (pri->code_buf));
+	if (arg & __FLXPATH)
+		(void) strlcat(str, "|__FLXPATH", sizeof (pri->code_buf));
```

Comment:
This one is slightly larger and uses `strlcat` so it seems safer than the pfiles one.

Reply:
Same question(s) apply, however.